torchtune/datasets/_text_completion.py

```diff
@@ -52,6 +56,9 @@ def __init__(
         self._column = column
         self.add_eos = add_eos
 
+        if filter_fn:
```

Comment:
Super nit at the 11th hour, I'm sorry:

```suggestion
        if filter_fn is not None:
```